tfe/resource_tfe_workspace.go

```diff
@@ -194,56 +200,11 @@ func resourceTFEWorkspaceCreate(d *schema.ResourceData, meta interface{}) error
 func resourceTFEWorkspaceRead(d *schema.ResourceData, meta interface{}) error {
 	tfeClient := meta.(*tfe.Client)
 
-	// Get the organization and workspace name.
-	organization, name, err := unpackWorkspaceID(d.Id())
-	if err != nil {
-		return fmt.Errorf("Error unpacking workspace ID: %v", err)
-	}
-
-	log.Printf("[DEBUG] Read configuration of workspace: %s", name)
-	workspace, err := tfeClient.Workspaces.Read(ctx, organization, name)
+	id := d.Id()
```

Comment:
This is perfectly valid, but it's also very common and fine to just call `d.Id()` everywhere the ID is needed.